models.py

```diff
@@ -650,9 +650,13 @@ def forward(self, x, mask = None):
 			if self.legacy:
 				assert self.track_running_stats is False
 				# BUG: https://github.com/pytorch/pytorch/issues/44636
-				# std, mean = torch.std_mean(x, dim = -1, keepdim = True)
+				#std, mean = torch.std_mean(x, dim = -1, keepdim = True)
 				# replaced with new version for .onnx export fix!
-				std, mean = torch.std(x, dim=-1, keepdim=True), torch.mean(x, dim=-1, keepdim=True)
+				# BUG failed:Type Error: Type parameter (T) bound to different types (tensor(float16) and tensor(float) in node (Mul_88). Mul_88 node located in torch.std function.
+				#std = torch.std(x, dim=-1, keepdim=True)
+				mean = torch.mean(x, dim=-1, keepdim=True)
+				xlen = x.size(dim=-1)
+				std = ((x - mean).pow(2).sum(dim=-1, keepdim=True) / xlen).sqrt()
```

Comment:
`torch.std` uses Bessel unbiased estimate by default, your code performs biased estimation. This is not a big deal, but since you're doing this to imitate torch.std, could you please also use `xlen - 1` in denominator and use `clamp(min = 1)` to avoid division by zero? At the very least, please add a note about this.